CHANGES_NEXT_RELEASE

```diff
@@ -4,3 +4,26 @@
 - Fix: fix issue with unrecognized NGSILD-Tenant and NGSILD-Path headers
 - Fix: mongodb dependency moved to devDependencies (as it is used only in test code)
 - Upgrade mongodb dep from 3.6.12 to 4.7.0
+- Upgrade body-parser dependency from 1.19.0 to 1.19.1
+- Upgrade express dependency from 4.16.4 to 4.17.2
+- Upgrade got dependency from 11.8.2 to 11.8.3
+- Upgrade moment-timezone dependency from 0.5.25 to 0.5.34
+- Upgrade mongoose dependency from 5.7.7 to 5.13.14
+- Upgrade mu2 dependency from 0.5.20 to 0.5.21
```

Comment:
mu2 is not a dependency...

```suggestion
```

Reply:
I missed the removal of that in the text.